orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -36,6 +36,7 @@ import static com.netflix.spinnaker.orca.ExecutionStatus.SUCCEEDED
 @Slf4j
 class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask implements RetryableTask {
   static final String REFRESH_TYPE = "ServerGroup"
+  static final String REFRESH_CONTROLLERS_TYPE = "ControllerSet"
```

Comment:
Why can't the "controllerset" type accept refreshes for server groups instead of adding this new type here?

Reply:
We talked abou this.  I thought that you would like to break into two different request call.

Reply:
No maybe we miscommunicated, clouddriver should handle the single on demand update for any controller that can, which may be more than one.